docs/en/development-guide.md

```diff
@@ -119,3 +119,30 @@ To automatically fix markdownlint issues:
 ```bash
 make lint-markdown-fix
 ```
+
+## Releasing container image
+
+KHI automates the container image deployment process.
+After creating a dedicated tag by creating a release on GitHub, the container will be built automatically and pushed on the repository.
+These tag creations are restricted only for our repository admins.
+
+* Pre-release
+  * Name tag with `vx.y.z-beta` then it will be deployed at the following addresses:
+    * `asia.gcr.io/kubernetes-history-inspector/release:beta`
+    * `asia.gcr.io/kubernetes-history-inspector/release:vx.y.z-beta`
+* Release
+  * Name tag with `x.y.z` then it will be deployed at the following address:
```

Comment:
Here doesn't have `v`. Is this intended?

Reply:
Fixed! Thank you for catching this.